src/newViews/AnalyticalFramework/types.ts

```diff
@@ -178,10 +178,12 @@ export interface DateRangeWidget extends BaseWidget {
 export interface Matrix1dWidget extends BaseWidget {
     type: 'matrix-1d';
     data: Matrix1dData;
+    width: 'full';
 }
 export interface Matrix2dWidget extends BaseWidget {
     type: 'matrix-2d';
     data: Matrix2Data;
+    width: 'full';
```

Comment:
Isn't `widget` already defined in BaseWidget.
I don't think we need to restrict width to be full on matrices

Reply:
This is to add restrictions. Let's have the restriction, for now, we can make it lax later on.